docs/csharp/language-reference/operators/stackalloc.md

```diff
@@ -49,7 +49,10 @@ The amount of memory available on the stack is limited. If you allocate too much
 
 - Avoid using `stackalloc` inside loops. Allocate the memory block outside a loop and reuse it inside the loop.
 
-The content of the newly allocated memory is undefined. You should initialize it before the use. For example, you can use the <xref:System.Span%601.Clear%2A?displayProperty=nameWithType> method that sets all the items to the default value of type `T`.
+The content of the newly allocated memory is undefined. You should initialize it, either with a `stackalloc` initializer, or a method like <xref:System.Span%601.Clear%2A?displayProperty=nameWithType> before the use.
+
+> [!IMPORTANT]
+> Not initializing memory allocated by `stackalloc` is an important difference from the `new` operator. Memory allocated using the `new` operator is initialized to 0.
```

Comment:
```suggestion
> Not initializing memory allocated by `stackalloc` is an important difference from the `new` operator. Memory allocated using the `new` operator is initialized to the default value of type `T` (e.g. `0` for numeric types, `null` for reference types).
```
?
My concern is about `initialized to 0`...reference types should either be mentioned or excluded explicitely.

Reply:
LGTM. Thanks for this alternative.